app/webpacker/styles/welcome-guide.scss

```diff
@@ -751,7 +752,7 @@ $z-indices: ("bottom": 5, "low": 10, "medium": 15, "high": 20, "top": 25);
 
       .text {
```

Comment:
not sure if this is the right bit of css - but the 'You're not the only one with questions is slightly out of kilter with the other headings on the Welcome guide page as it is full width on mobile and none of the other headings are

however, this is actually like this in prod so happy to leave as is if necessary

![image](https://user-images.githubusercontent.com/87642394/196985089-2c8a6943-f048-4b02-a11a-aa8a0c347825.png)